### PR TITLE
test: custom postcss config

### PR DESCRIPTION
**Pull request by clarkdo**

### CODE REVIEW

File: test/fixtures/basic/nuxt.config.js [hunk 1]
```diff
@@ -15,5 +15,10 @@ module.exports = {
     bad: null,
     '': true
   },
-  transition: false
+  transition: false,
+  build: {
+    postcss: [
+      require('postcss-cssnext')()
```
Comment (pi0) [inline test/fixtures/basic/nuxt.config.js]: Maybe would be better if we use a different plugin like rtlcss for tests. As cssnext is also in default configs so tests will fail even if this option totally being ignored! And resulting into false-positive coverage ;)

Comment (clarkdo) [reply test/fixtures/basic/nuxt.config.js]: yes, that's a good idea, will refactor it. 😄

Comment (pi0) [reply test/fixtures/basic/nuxt.config.js]: Indeed that just was a point so maybe not a priority thing. Thanks mate!

Comment (clarkdo) [reply test/fixtures/basic/nuxt.config.js]: ok ok, I will do that when time permits, thanks for the reviewing, bro 😄

**Pull request merged**